test/parallel/test-tls-keylog-tlsv13.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const events = require('events');
+const tls = require('tls');
+const fixtures = require('../common/fixtures');
+
+async function main() {
```

Comment:
I would prefer this test to be written in vanilla js, for readability. Also, I suspect it could be much shorter if it used the https://github.com/nodejs/node/blob/master/test/fixtures/tls-connect.js fixture, grep test/parallel for examples.

Reply:
@sam-github I find the async/await version a lot more readable than I imagine a callback-based version to be

Reply:
I have changed the test to use callbacks, and it turned out to be readable IMHO. Let me know what you think.

@sam-github I looked at that fixture, but it doesn't seem appropriate because I need to subscribe to `keylog` just after the TLSSocket is created (i.e. before the handshake).

Reply:
Right, I see your point, the fixture doesn't allow event setup. :+1: